oraichain/assetlist.json

```diff
@@ -718,6 +716,36 @@
           }
         }
       ]
+    },
+    {
+      "description": "The PEPE native token of Oraichain",
+      "denom_units": [
+        {
+          "denom": "factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/extPEPE",
+          "exponent": 0
+        },
+        {
+          "denom": "pepe",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/extPEPE",
+      "name": "Pepe",
+      "display": "pepe",
+      "symbol": "PEPE",
```

Comment:
How did Ethereum PEPE get here? Is it bridged or a synthetic?

Reply:
They are bridged over oraibridge (obridge-sdk) which i have not registered on chain-registry.

Reply:
I'm thinking it might be as simple as adding a trace to pepe, similar to what was added for Oraichain's TON:
https://github.com/oraichain/chain-registry-fork/blob/feat/add-pepe-oraichain/oraichain/assetlist.json#L686-L693